studio/studio.py

```diff
@@ -291,11 +465,91 @@ async def serve(q: Q):
                 editor.update_file_tree(q, project.dir)
                 await q.page.save()
                 editor.open_file(q, project.entry_point)
+                requirements_file = open('project/requirements.txt', 'r') if os.path.exists(
+                    'project/requirements.txt') else None
+                if requirements_file:
+                    await show_packages_dialog(q)
+                    q.client.task = asyncio.create_task(pip(
+                        q,
+                        'install',
+                        '-r',
+                        'project/requirements.txt',
+                        None,
+                        'Installing packages from requirements.txt',
+                        f'Packages from requirements.txt installed successfully.',
+                        f'Error installing requirements.txt.',
+                        None,
+                        on_requirements_install_error
+                    ))
             else:
                 q.page['meta'].dialog.items = [
                     ui.message_bar(type='error', text='There must be exactly 1 root folder.'),
                     ui.button(name='import_project', label='Import again', icon='Upload'),
                 ]
+    elif q.args.show_packages_dialog:
+        await show_packages_dialog(q)
+    elif q.events.package_dialog and q.events.package_dialog.dismissed:
+        q.page['meta'].dialog = None
```

Comment:
There is always going to be just one dialog so no need to distinguish between `package_dialog` and `dialog` in terms of events. WIll make code simpler as there would be no need for separate handlers.